### PR TITLE
Updating AzureGermanCloud AD Endpoint

### PR DESCRIPTION
**Pull request by JeffSimmer**
Removing an extra '/' from the AzureGermanCloud
activeDirectoryEndpointUrl. The extra '/' causes an exception when
logging in to a subscription in the AzureGermanCloud environment.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@JeffSimmer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jesimmer). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by amarzavery** [comment]
https://github.com/Azure/azure-xplat-cli/issues/2787

File: lib/util/profile/environment.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ Environment.publicEnvironments = [
     sqlManagementEndpointUrl: 'https://management.core.cloudapi.de:8443/',
     sqlServerHostnameSuffix: '.database.cloudapi.de',
     galleryEndpointUrl: 'https://gallery.cloudapi.de/',
-    activeDirectoryEndpointUrl: 'https://login.microsoftonline.de/',
```
Comment (amarzavery) [inline lib/util/profile/environment.js]: @JeffSimmer - Can you also remove the trailing slash from managementEndpointUrl, resourceManagerEndpointUrl, while you are at it. That should resolve the next set of errors that people may encounter after successful login.

Thanks for the contribution.

Comment (JeffSimmer) [inline lib/util/profile/environment.js]: Sure, I'll remove those as well.

**Pull request merged**